packages/web/src/components/track/MetadataItem.tsx

```diff
@@ -0,0 +1,20 @@
+import { PropsWithChildren } from 'react'
+
+import { Flex, Text } from '@audius/harmony'
+
+type MetadataItemProps = PropsWithChildren<{
+  label: string
+}>
+
+export const MetadataItem = ({ label, children }: MetadataItemProps) => {
```

Comment:
guessing this shouldnt be in "track" space? i think we have an "entity" directory?

Reply:
there was some dead code in `entity-page`, repurposed that